lib/bamboozled/api/base.rb

```diff
@@ -35,7 +35,11 @@ def request(method, path, options = {})
           case response.code
           when 200..201
             begin
-              JSON.parse(response).with_indifferent_access
+              if response.nil?
+                {"headers" => response.headers}.with_indifferent_access
```

Comment:
@Enriikke here is a gist with different things I've tried and what they return, https://gist.github.com/kylefdoherty/9927a290f0b998298e18.

When you don't include anything under the headers in the json file `response.body` is nil.  However, I can do `response.body.to_s` to convert the nil value to an empty string.  This isn't necessary when actually using the API (a successful POST returns an empty string) but will get the test to pass.  I don't love this solution but can't figure out a way to have the stubbed response return an empty string for `response.body`